[Edit & New Chapter] Alternative models of culture change! (#3941)
the-turing-way/the-turing-way: book/website/ethical-research/cultural-change/alt-models-discussion.md

The explicit hierarchy can also be interpreted as assigning relative importance to each of the stages, when often their relative importance fluctuates depending on the norms, practices, existing infrastructure, and the influence that each of these factors exerts on the other. 
Although the blog post in which the model is introduced offers more context about why specifically, the authors have chosen to present their model for change in this way, we now frequently see the diagram taken out of context and used to justify techno-solutionism.
This further underscores the need for advocates to think critically about designing visual representations and how they may be misinterpreted or used out of context.
This might need to be balanced against the effectiveness of communicating concepts using simple structures and clear diagrams

(er-cc-am-discussion-factors)=
## Factors to consider in culture change modelling

> So, if we reject the hierarchical perspective presented in Nosek et al in our discussion, what are the factors we feel need to be considered when it comes to culture change in research practices?

Reviewer: ```suggestion
So, if we reject the hierarchical perspective presented by Nosek in our discussion, what are the factors we feel need to be considered when it comes to culture change in research practices?
```

Reviewer: use of et al + I think it is just Nosek's name on the blog?

> The group discussed several broad groups of factors, all of which 

Reviewer: all of which what? :D

### The past 
To be human is to exist with biases and limited perspectives. 
Those advocating for cultural change will find they need to push to address resistance borne out of people's previous experiences, historical preferences, and the ease of doing things the way they have always been done. 

### Aligning systems and behaviour 
Systems and behaviour often need to be tackled in parallel. 
It's hard to change someone's behaviour if the system is not capable of supporting the change; and changing the system does not automatically change people's behaviours (see open access mandates).

> They may not match perfectly at first, but it critical to acknowledge and address both if you are seeking long-term change.

Reviewer: ```suggestion
They may not match perfectly at first, but it is critical to acknowledge and address both if you are seeking long-term change.
```

### Taking care 
Change is often hard for people, even when they can clearly see the need for it. 
People can be risk adverse, and manage this by sticking with what they know already has worked in the past. 
Building the case for change, acknowledging the potential impacts and working to mitigate them, and employing specific tactics such as change champions, can all help.
It is also important to hold space to discuss changes, get people excited about change, and incorporate their insights as the process proceeds.

### Acknowledging power structures
The hierarchical, rigid power structures of the academic system are well known, and despite some efforts, remain relatively entrenched.
Those more senior in the current system have benefited from it, and may be less motivated to support and empower those seeking change. 

> The level of influence and empowerment individuals have to effect the culture change they want to see in the world may be limited by their seniority, or perceived expertise. 

Reviewer: ```suggestion
The level of influence and empowerment individuals have to affect the culture change they want to see in the world may be limited by their seniority, or perceived expertise. 
```

This also intersects with systemic biases and injustices such as racism, sexism, and ableism, and in some cases magnifies them. 
This can be tackled with collective organising and action, but this also takes time to build, manage, and sustain. 

(er-cc-am-discussion-models)=
## Alternative models of change? 
From the group discussion we came up with a number of different representations of research culture change approaches:

* A buffet of culture change factors, with sizing according their influence in specific situations, avoiding the implicit and explicit hierarchy of other models.
* An expanded pyramid with more feedback loops and other factors feeding into each level to demonstrate the complexity of change.

> * A 4-way matrix between human infrastructure, technical infrastructure, community needs and problems, and societal infrastructure, all interconnected and feeding back on each other. 

Reviewer: ```suggestion
* A four-way matrix between human infrastructure, technical infrastructure, community needs and problems, and societal infrastructure, all interconnected and feeding back on each other. 
```
